.docker/public-nginx.conf

```diff
@@ -47,3 +47,18 @@ server {
     }
 }
 
+server {
+    server_name ~^.+--pattern-library\.libero\.pub$;
+    listen 80;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt { access_log off; log_not_found off; }
+
+    location / {
+        proxy_pass http://pattern_library:80;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Port $server_port;
+        proxy_set_header X-Forwarded-Proto $scheme;
```

Comment:
standard boilerplate but is it actually used? Maybe in some nginx logs?

Reply:
Probably not since it's a static site. Just going for correctness (this level shouldn't know the internals).